examples/solvers/functional_basic_example.py

```diff
@@ -4,11 +4,7 @@
 information on functionals, see `the ODL functional guide
 <http://odlgroup.github.io/odl/guide/in_depth/functional_guide.html>`_
 """
-
-# Imports for common Python 2/3 codebase
-from __future__ import print_function, division, absolute_import
-from future import standard_library
-standard_library.install_aliases()
+from __future__ import print_function, division
```

Comment:
There are no parens when you do this in py2:

```python
>>> print(2)
2
```

It only occurs when you use commas:

```python
>>> print(2, 3)
(2, 3)
```

which differs from python 3:

```python
>>> print(2, 3)
2 3
```